src/main/java/com/google/cloud/tools/gradle/appengine/standard/AppEngineStandardPlugin.java

```diff
@@ -101,14 +102,27 @@ private void configureExtensions() {
             deploy.setAppEngineDirectory(
                 new File(stageExtension.getStagingDirectory(), "WEB-INF/appengine-generated"));
           }
-          deploy.setProjectResolver(
-              new ProjectResolver(
-                  stageExtension
-                      .getStagingDirectory()
+
+          File appengineWebXml =
```

Comment:
I was basing the logic of of `AppEnginePlugin#isAppEngineStandard`, it looks like there's a fallback.

Reply:
ah yes, good that you brought that up, it's because it's possible at that point the "war" plugin hasn't resolved yet. For instance a project buildfile like this
```
apply plugin "com.google.cloud.tools.appengine"
apply plugin "war"
```
but I think at the point you're resolving it, it shouldn't be a problem, since it's after all plugins have been applied.

Reply:
Ah ok, makes sense.